src/treemodel/nodelist.js

```diff
@@ -293,7 +298,7 @@ export default class NodeList {
 
 		this._indexMap.splice( index, number );
 
-		for ( let i = index; i < this._indexMap.length ; i++ ) {
+		for ( let i = index; i < this._indexMap.length; i++ ) {
```

Comment:
This change is not part of this ticket. Please report a separate ticket for it.






trollolololo ;P